ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -167,7 +230,218 @@ export default class SecurityTab extends PureComponent {
     );
   }
 
-  renderMultiAccountBalanceCheckerOptIn() {
+  renderShowGasFeeEstimationBuySwapTokensToggle() {
```

Comment:
Let's remove this from this PR. We will add this in a future PR, when we are ready to deal with those features being toggled off.